storage/index.md

```diff
@@ -70,38 +70,117 @@ PostgreSQL have some multi-modality capability.]
 | PostreSQL     |          | x            |
 | Cassandra     | x        |              |
 | Clickhouse    | x        |              |
-
-
--- write something about how columnar formats are written in some
-   order, and being cognizant of the order that the data, so that you
-   can use the order of the records in the parquet file as an index
-   for ORDER BY and GROUP BY statements.
+| DuckDB        | x        |              |
```

Comment:
Does GlareDB support DuckDB? I thought we hadn't added that yet and I don't see any tests for it.

It looks like Snowflake and BigQuery are missing from this list, as is Iceberg.

Reply:
GlareDB doesn't support DuckDB, but it also doesn't support avro? This isn't "things we support" just an overview of the formats, but we can also change however we want.

Reply:
The intro paragraph says:

> This table outlines some of the data formats and sources **that GlareDB
supports** and identifies the modality. [n.b. Engines like MySQL and
PostgreSQL have some multi-modality capability.]

Should it just "outlines some data formats and sources"?

Reply:
My 2c here is that it makes sense to either omit items we don't support, or asterisk those items as planned

Reply:
I added a support as a column

Reply:
I removed the things